Spanish – Many translation updates

cmuncharaz opened the pull request:
<!--Describe the pull request below.-->
- Updated the Spanish translation of the following pages:
    - W3C Accessibility Standards Overview.
    - WCAG 2 Overview.
    - WCAG 2 at a Glance.
    - Design and Develop Overview
    - Evaluating Web Accessibility Overview
    - The Business Case for Digital Accessibility
    - Teach and Advocate Overview
- Amended an error in the changelog page.

<!--If your PR has a primary page for review, set an entry path.
    Add a relative path next to `@netlify` below.-->

@netlify /standards-guidelines/es

remibetin commented on pages/standards-guidelines/changelog.md:
```diff
@@ -26,7 +26,7 @@ footer: >
   - updated the link to “How to Meet WCAG 2” to target the WCAG 2.2 version: `https://www.w3.org/WAI/WCAG22/quickref/`.
   - reordered the “WCAG 2.x Standard” list items to start with WCAG 2.2.
-- In the "Authoring Tool Accessibility Guidelines (ATAG)" section: updated a link to target the most recent WAI-ARIA 1.2 standard.
+- In the "Accessible Rich Internet Applications (WAI-ARIA)" section: updated a link to target the most recent WAI-ARIA 1.2 standard.
```
@cmuncharaz Thanks for spotting this!

cmuncharaz commented:
Updated the Spanish translation of Design and Develop Overview page.

cmuncharaz commented on pages/standards-guidelines/index.es.md:
```diff
@@ -4,11 +4,11 @@
+last_updated: 2024-02-29  # Keep the date of the English version
 
 translation:
   status: outdated
```
@remibetin Should I update the status?

remibetin replied:
@cmuncharaz Yes, you can remove the `status: outdated` line.

cmuncharaz commented:
- Removed the translation outdated status from the pages included in this pull request.
- Updated the Spanish translation of the following pages:
    - Evaluating Web Accessibility Overview
    - The Business Case for Digital Accessibility
    - Teach and Advocate Overview

remibetin commented on pages/standards-guidelines/wcag/glance.es.md:
```diff
@@ -5,11 +5,11 @@
 title: "WCAG 2.1 de un vistazo"
```
```suggestion
title: "WCAG 2 de un vistazo"
```

remibetin commented on pages/standards-guidelines/wcag/glance.es.md:
```diff
@@ -5,11 +5,11 @@
-last_updated: 2018-06-05  # Keep the date of the English version
+last_updated: 2023-11-29  # Keep the date of the English version
+first_published: "Julio del 2008"
 description: Esta página proporciona un resumen no literal de las Pautas de Accesibilidad para el Contenido Web (WCAG) 2.1.
```
```suggestion
description: Esta página proporciona un resumen no literal de las Pautas de Accesibilidad para el Contenido Web (WCAG) 2.
```

remibetin commented on pages/standards-guidelines/wcag/glance.es.md:
```diff
@@ -41,24 +40,26 @@ footer: >
 {% include box.html type="end" %}
 {:/}
 
 Por favor, consulte los siguientes recursos clave para aprender a usar WCAG 2.1:
```
```suggestion
Por favor, consulte los siguientes recursos clave para aprender a usar WCAG 2:
```

remibetin commented on pages/standards-guidelines/wcag/glance.es.md:
```diff
@@ -5,11 +5,11 @@
 
 translation:
-  status: outdated
   last_updated: 2019-07-03  # Put the date of this translation YYYY-MM-DD (with month in the middle)
```
@cmuncharaz Please update `last_updated`.

remibetin commented on pages/design-develop/index.es.md:
```diff
@@ -5,10 +5,9 @@
 
 translation:
-  status: outdated
   last_updated: 2019-09-30  # Put the date of the translation YYYY-MM-DD (with month in the middle)
```
@cmuncharaz Please update `last_updated`.

remibetin reviewed (changes requested):
Hi @cmuncharaz,

Thank you very much for your contributions.

I have added a few comments.

remibetin commented:
Thanks @cmuncharaz for these updates!

I fixed a few omitted substitutions of "WCAG 2.1" with "WCAG 2" (see commits and resolved comments above).
